src/inc/woocommerce-actions.php

```diff
@@ -40,6 +40,8 @@ public static function hooks() {
 		add_action( 'woocommerce_remove_cart_item', array( static::class, 'remove_from_cart' ), 100, 2 );
 		add_action( 'woocommerce_update_order', array( static::class, 'update_order' ), 100, 2 );
 		add_action( 'woocommerce_order_applied_coupon', array( static::class, 'add_promotion' ), 100, 2 );
+		add_action( 'sift_for_woocommerce_chargeback', array( __CLASS__, 'chargeback' ), 100, 3 );
+		add_action( 'woocommerce_payments_before_webhook_delivery', __NAMESPACE__ . '\process_before_event_delivery', 100, 2 );
```

Comment:
This should be loaded only for WooPayments (even if it makes sense to have it globally)